packages/rlc-common/CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.38.5 (2025-03-24)
+
+- [Feature] Improve generateTest config. Please refer to [#3125](https://github.com/Azure/autorest.typescript/pull/3125)
+- [Feature] Support tsconfig changes, vitest config changes and api-extractor changes from migration tool for codegen. Please refer to [#3081](https://github.com/Azure/autorest.typescript/pull/3081)
```

Comment:
```suggestion
- [Feature] Support esm generation. Please refer to [#3081](https://github.com/Azure/autorest.typescript/pull/3081)
```

Reply:
Can you also add one feature for generate module as ESM and list other related items as details?